cpp/src/parquet/properties.h

```diff
@@ -941,6 +942,15 @@ class PARQUET_EXPORT ArrowReaderProperties {
     return coerce_int96_timestamp_unit_;
   }
 
+  /// Enable Parquet supported Arrow Extension Types.
+  ///
+  /// When enabled, Parquet will use supported Arrow ExtensionTypes in mapping to Arrow
+  /// schema. Currently only arrow::extension::json() extension type is supported. This
+  /// will be used for utf8 columns whose LogicalType is JSON.
+  void set_arrow_extensions_enabled() { arrow_extensions_enabled_ = true; }
+  void set_arrow_extensions_disabled() { arrow_extensions_enabled_ = false; }
+  bool get_arrow_extensions_enabled() const { return arrow_extensions_enabled_; }
```

Comment:
I don't know whether get/set is proper...

Reply:
Changed.